src/apify/_actor.py

```diff
@@ -513,6 +547,19 @@ async def set_value(
         key_value_store = await self.open_key_value_store()
         return await key_value_store.set_value(key, value, content_type=content_type)
 
+    def get_charging_manager(self) -> ChargingManager:
+        """Retrieve the charging manager to access granular pricing information."""
+        self._raise_if_not_initialized()
+        return self._charging_manager
+
+    async def charge(self, event_name: str, count: int = 1) -> ChargeResult:
+        """Charge for a specified number of events - sub-operations of the Actor.
+
+        This is relevant only for the pay-per-event pricing model.
+        """
+        self._raise_if_not_initialized()
+        return await self._charging_manager.charge(event_name, count)
```

Comment:
missing arg docstring

Reply:
True, but what useful information can I provide there?

Reply:
1) it will be rendered as an empty description field otherwise
2) https://github.com/apify/apify-sdk-python/pull/393#pullrequestreview-2626102302